src/test/v1.2/basic/fdc3.joinChannel.ts

```diff
@@ -27,7 +27,6 @@ export default () =>
     });
 
     it("(BasicJC1) Can join channel and broadcast", async () => {
```

Comment:
title should be "(BasicJC1) Can join channel"

Reply:
fixed, thanks.